### DISCUSSION TITLE
Release V1.0.13

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#4664] [Polls]
https://github.com/JanssenProject/jans/milestone/16

**Comment by manojs1978** [comment]
blocker
https://github.com/GluuFederation/flex/issues/998
https://github.com/JanssenProject/jans/issues/4847

**Reply by manojs1978** [reply]
agama flow not working
https://github.com/JanssenProject/jans/issues/4875

**Reply by manojs1978** [reply]
agama flow working fine
